dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/organisationunit/hibernate/HibernateOrganisationUnitGroupStore.java

```diff
@@ -65,4 +67,15 @@ public HibernateOrganisationUnitGroupStore( SessionFactory sessionFactory, JdbcT
     {
         return getQuery( "from OrganisationUnitGroup g where size(g.groupSets) = 0" ).list();
     }
+
+    @Override
+    public OrganisationUnitGroup getOrgUnitGroupInGroupSet( Set<OrganisationUnitGroup> groups,
+        OrganisationUnitGroupSet groupSet )
+    {
+        return getQuery(
+            "select g from OrganisationUnitGroup g inner join g.groupSets gs where gs = :groupSet and g in :groups" )
+                .setParameter( "groupSet", groupSet )
+                .setParameter( "groups", groups )
```

Comment:
I think previously we had to use `setParameterList` with IDs but good to see it now works with just `setParameter`.

Reply:
yes, hibernate will auto detect if value is collection then will call setParameterList